src/mesh_motion/MotionRotation.C

```diff
@@ -40,13 +40,16 @@ void MotionRotation::load(const YAML::Node& node)
 }
 
 void MotionRotation::build_transformation(
-  const double time,
+  double time,
   const double*  /* xyz */)
 {
   double eps = std::numeric_limits<double>::epsilon();
 
-  if( (time >= (startTime_-eps)) && (time <= (endTime_+eps)) )
+  if(time >= (startTime_-eps))
   {
+    if(time >= (endTime_+eps))
+      time = endTime_;
```

Comment:
You are modifying a variable passed by value here. 1. What is the use case for modifying a variable like `time` within this method? and 2. pass by value variables won't have an effect outside of this method.

Reply:
1) The use case is - If end time for a particular motion is less than the total simulation time. In such a case the mesh configuration for time > end_time should correspond to when time = end_time.
 
2) Following 1), the variable is meant to reflect the change only within this function.

Reply:
@sayerhs I have addressed the coding style issues in b05fabab6b4cbc4bcea938a616419fa8b34dd792